source/docs/blockchain_basic_configuration.md

```diff
@@ -0,0 +1,66 @@
+title: Blockchain configuration
+---
+
+When in development phase, it is necessary to use a local Ethereum node to deploy our contracts to, either using a simulator or client like Geth or Parity.
```

Comment:
Theoretically, having a node set up for Embark is actually optional. So "necessary" might not be the right wording here. How about something like this:

> If we decide to build a decentralized application that should be powered by a blockchain, we need to tell Embark what node it should connect to, to deploy our application accordingly. This can be either a [blockchain simulator]() (link to simulator section in running apps guide), or a real node like [Geth]() or [Parity]().
>
> In this guide we'll explore how to configure the blockchain node we want Embark to connect to.